pkg/events/derive/derive.go

```diff
@@ -42,20 +42,23 @@ func (t Table) Register(deriveFrom, deriveTo events.ID, deriveCondition func() b
 
 // DeriveEvent takes a trace.Event and checks if it can derive additional events from it as defined by a derivationTable.
 func (t Table) DeriveEvent(event trace.Event) ([]trace.Event, []error) {
-	derivatives := []trace.Event{}
-	errors := []error{}
+	derivedEvents := []trace.Event{}
+	deriveErrors := []error{}
 	deriveFns := t[events.ID(event.EventID)]
 	for id, deriveFn := range deriveFns {
 		if deriveFn.Enabled() {
-			derivative, errs := deriveFn.DeriveFunction(event)
+			derivs, errs := deriveFn.DeriveFunction(event)
 			for _, err := range errs {
-				errors = append(errors, deriveError(id, err))
+				deriveErrors = append(deriveErrors, deriveError(id, err))
+			}
+			for _, deriv := range derivs {
+				deriv.PoliciesVersion = event.PoliciesVersion
```

Comment:
I think this should be done at [`buildDerivedEvent`](https://github.com/aquasecurity/tracee/blob/main/pkg/events/derive/derive.go#L130)

Reply:
Ok, as long as it is a function accessible via policyManager.